src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -2031,17 +2045,19 @@ def cli(self, service, output=None):
         # Cache - Remote EID mapping cache size, IID - Instance ID,
         # Role - Configured Role
 
-        #                       Interface    DB  DB no  Cache  Incom  Cache 
+        #                       Interface    DB  DB no  Cache  Incom  Cache
         # EID VRF name             (.IID)  size  route   size  plete  Idle  Role
         # red                   LISP0.101     1      0      2   0.0%  0.0%  ITR-ETR
         # blue                  LISP0.102     1      0      1   0.0%    0%  ITR-ETR
+        # blue                  LISP0.102@    1      0      1   0.0%    0%  ITR-ETR
+        # blue                  LISP0.102*    1      0      1   0.0%    0%  ITR-ETR
         p4_1 = re.compile(r'(?P<vrf>(\S+)) +(?P<interface>(\S+))\.(?P<iid>(\d+))'
-                         ' +(?P<db_size>(\d+)) +(?P<db_no_route>(\d+))'
+                         ' +(?P<db_size>(\d+))(?P<rloc_status>([@*]?)) +(?P<db_no_route>(\d+))'
```

Comment:
like 'encap_to_petr', if possible, please update schema to accomodate 'rloc_status' for this parser.

Reply:
OK, should I make a new push req?

Reply:
yes, please push new change to this PR.